app/src/androidTest/java/com/duckduckgo/app/fire/AuthDatabaseLocatorTest.kt

```diff
@@ -16,17 +16,31 @@
 
 package com.duckduckgo.app.fire
 
+import android.content.Context
 import androidx.test.platform.app.InstrumentationRegistry
+import com.duckduckgo.app.browser.DefaultWebViewDatabaseProvider
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
 import org.junit.Assert.*
+import org.junit.Before
 import org.junit.Test
 
+@ExperimentalCoroutinesApi
 class AuthDatabaseLocatorTest {
 
+    private val context: Context = InstrumentationRegistry.getInstrumentation().targetContext
+
+    @Before
+    fun before() {
+        DefaultWebViewDatabaseProvider(context).get()
+
+        // Wait for the database path to be created
```

Comment:
I don't have the full context here, but could we check for the existence of the "database path" ourselves and wait if it's not yet created (instead of the arbitrary sleep)?